### PR TITLE
added search members feature

### LINKED ISSUE #164: Search / Sort members on parameters
We want to sort users:

- Alphabetically (First name)
- Based on skills: 'NodeJS', 'ReactJS', 'JS'
- Based on accomplishments levels: 'Send proper PRs', 'Code reviewer', 'Conducted KT session', 'Leader', 'Feature Owner', etc
- Based on recently active in tasks
- Open to new roles

### PR DESCRIPTION
**Pull request by RohitKuwar**
### What is the change?
Closes #164
Now we have a search box on the website. If you want to search for a member, you can type the member's name inside the search box and you will get a particular member's card.

### Is it bug?

- No. It is a feature.

### \*Dev Tested?

- [x] Yes
- [ ] No

#### Platforms

- [x] Web

#### Browsers

- [x] Chrome
- [ ] Safari
- [x] Firefox

### Before / After Change Screenshots:

https://user-images.githubusercontent.com/70262783/152112092-431637dd-46f4-4c9b-ac11-7c48267b6022.mp4

### CODE REVIEW

File: src/components/UI/search-box/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import { searchMemberContext } from '@store/search-members/searchMembers-context';
+import classNames from './search-box.module.scss';
+
+const SearchBox = () => {
+  const { setSearchTerm } = searchMemberContext();
+  return (
+    <div className={classNames.searchContainer}>
+      <input
+        type="text"
+        placeholder="&#128269; search members here"
```
Comment (sumitd94) [inline src/components/UI/search-box/index.js:11]: Can we please get this value from constants?
I think we have a constants file - https://github.com/Real-Dev-Squad/website-members/blob/develop/src/constants/AppConstants.js

File: src/helper-functions/search-members.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const searchMembers = (membersArr, searchTerm) => {
+  return membersArr.filter(
```
Comment (harshith-venkatesh) [inline src/helper-functions/search-members.js:2]: Should we be putting a check on searchTerm before proceeding with filtering?
Does it equally work the same with empty string too?

Comment (RohitKuwar) [reply src/helper-functions/search-members.js:2]: Yes, we are putting a check on searchTerm before proceeding with filtering. When it is empty string members array is at its original position. When we type something then it finds the string in the array and we get a filtered array.

File: src/components/UI/search-box/index.js (hunk 1 above)
Comment (RohitKuwar) [reply src/components/UI/search-box/index.js]: Done. Please check

**Review by sumitd94 — APPROVED** [review]

**Review by harshith-venkatesh — APPROVED** [review]

**Pull request merged**